src/commands/defaultSignImage.ts

```diff
@@ -0,0 +1,37 @@
+import * as vscode from 'vscode';
+import { logInToDockerCli } from './registries/logInToDockerCli';
+import { checkCLI } from '../utils/checkCLI';
+import { execAsync } from '../utils/execAsync';
+
+
+const cliName = 'notation';
+const goToNotationButton = 'Download Notation';
+const NotationURL = 'https://notaryproject.dev/docs/installation/cli/';
+const errorMessage = `Notation not found: `;
+
+//signs image with default key
+export async function defaultSign(imageTag: any): Promise<void> {
+    await logInToDockerCli(imageTag.parent);
+    var notaryCli = await checkCLI(cliName);
+    if(notaryCli) {
+            const progressOptions: vscode.ProgressOptions = {
+                location: vscode.ProgressLocation.Notification,
+                title: vscode.l10n.t('Signing...'),
+            };
+            await vscode.window.withProgress(progressOptions, async () => {
+                var errOut = await execAsync(`${cliName} sign ${imageTag.fullTag}`); //TODO: referrence image by digest not tag
```

Comment:
nit: this variable name is a bit confusing, since it implies this will always err.  Shouldn't it be more like "signSuccess" or something around those lines?